README.md

```diff
@@ -0,0 +1,348 @@
+  --reports all
+  --output robocop.log
+  ```
+
+  ---
+
+  ### `.robocop` file
+
+  It is a default file that is loaded only when no command line options are provided for Robocop.
+  When running plain `robocop` command, it looks for `.robocop` file from place where it was run
+  until it finds `.git` file. Options can be provided like in the example above.
+
+  ---
+
+  ### `pyproject.toml` file
+
+  If there is no `.robocop` file and `toml` module is installed,
+  Robocop will try to load configuration from `pyproject.toml` file (if it exists).
+  Options have the same names as command line arguments
+  and need to be placed under `[tool.robocop]` section.
+
+  Example configuration file:
+  ```commandline
+  [tool.robocop]
+  paths = [
+      "tests\\atest\\rules\\bad-indent",
+      "tests\\atest\\rules\\duplicated-library"
+  ]
+  include = ['W0504', '*doc*']
+  exclude = ["0203"]
+  reports = [
+      "rules_by_id",
+      "scan_timer"
+  ]
+  ignore = ["ignore_me.robot"]
+  ext_rules = ["path_to_external\\dir"]
+  filetypes = [".txt", ".csv"]
+  threshold = "E"
+  format = "{source}:{line}:{col} [{severity}] {rule_id} {desc} (name)"
+  output = "robocop.log"
+  configure = [
+      "line-too-long:line_length:150",
+      "0201:severity:E"
+  ]
+  no_recursive = true
+  ```
+</details>
+
+<details>
+  <summary>I use different coding standards. Can I configure rules so that they fit my needs?</summary>
+
+  **Yes**, some rules are configurable. You can list them by running `robocop --list-configurables`
+  or just `robocop -lc`.
+
+  Configuring is done by using `-c / --configure` command line option followed by pattern
+  `<rule>:<param_name>:<value>` where:
+  - `<rule>` can either be rule name or its id
+  - `<param_name>` is a public name of the parameter
+  - `<value>` is a desired value of the parameter
+
+  For example:
+  ```commandline
+  --configure line-too-long:line_length:140
+  ```
+  is equivalent to
+  ```commandline
+  -c 0508:line_length:140
+  ```
+  ---
+  Each rule's severity can also be overwritten. Possible values are
+  `e/error`, `w/warning` or `i/info` and are case-insensitive. Example:
+  ```commandline
+  -c too-long-test-case:severity:e
+  ```
+  ---
+  If there are special cases in your code that violate the rules
+  you can also exclude them in the source code. 
+
+  Example:
+  ```
+  Keyword with lowercased name  # robocop: disable
+  ```
+
+  More about it in
+  [our documentation](https://robocop.readthedocs.io/en/latest/including_rules.html#ignore-rule-from-source-code).
+
+</details>
+
+<details>
+  <summary>Can I define custom rules?</summary>
+
+  **Yes**, you can define and include custom rules using `-rules / --ext_rules` command line option
+  by providing a path to a file containing your rule(s). The option accepts comma-separated list
+  of paths to files or directories, e.g. 
+  ```robocop -rules my/own/rule.py --ext_rules rules.py,external_rules.py```.
```

Comment:
```commandline
...
```
otherwise it's displayed in the same line:
![image](https://user-images.githubusercontent.com/8532066/113816898-c4950380-9775-11eb-8fb6-1237ef7604b8.png)

Reply:
Changed